index.html

```diff
@@ -1,14 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="pages/index.css">
+    <meta charset="utf-8" >
```

Comment:
@Woolfs1423 Last thing, please remove those trailing spaces for lines from 4 to 6

```HTML
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<link rel="stylesheet" href="pages/index.css">
```